lib/hammer/backend.ex

```diff
@@ -1,41 +0,0 @@
-defmodule Hammer.Backend do
-  @moduledoc """
```

Comment:
why removing the behavior? we need one for other backedn type to ensure they implement they proper contract?

Reply:
I moved it into `Hammer` for now, but might move it into a separate module.